pkl-core/src/main/java/org/pkl/core/util/AnsiTheme.java

```diff
@@ -28,7 +28,7 @@ private AnsiTheme() {}
 
   public static final AnsiCode STACK_FRAME = AnsiCode.FAINT;
   public static final AnsiCode STACK_TRACE_MARGIN = AnsiCode.YELLOW;
-  public static final AnsiCode STACK_TRACE_LINE_NUMBER = AnsiCode.BLUE;
+  public static final AnsiCode STACK_TRACE_LINE_NUMBER = AnsiCode.BRIGHT_BLUE;
```

Comment:
How about `AnsiCode.FAINT`? The bright blue draws attention to the line number, but it's arguably one of the less important parts of the error message.

<img width="642" alt="Screenshot 2024-11-04 at 4 31 06 PM" src="https://github.com/user-attachments/assets/0baf33c2-b18e-460e-9547-c5ee433307d7">

Reply:
Looks even better (updated).

By the way, is there a reason why I can't set colors in `settings.pkl`?

Reply:
I haven't put too much thought into it so far. I don't see any reason why it shouldn't be there, though.